### PR TITLE
CFIN-209 Add parameter to Funnelback URL to request only courses

### PR DESCRIPTION
**Pull request by ncn1**
This successfully stops this page https://www.york.ac.uk/study/undergraduate/courses/teaching-learning/ being returned when  Archaeology is entered as the search term

### CODE REVIEW

File: src/constants/UrlAndParameters.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ const URL_AND_PARAMETERS = {
     COLLECTION: "york-uni-courses",
     FORM: "course-search",
     PROFILE: "_default_preview",
+    SMETA_CONTENT_TYPE: "course",
```
Comment (RichardAllum) [inline src/constants/UrlAndParameters.js:6]:
```suggestion
    CONTENT_TYPE: "course",
```
I think the name `smeta_contentType` is weird enough that it should also be a constant here, although at that point would it be better to have an object with default keys and values that the UrlConstructor loops through?

**Review by RichardAllum — APPROVED** [review]

File: src/constants/UrlAndParameters.js (hunk 1 above)
Comment (ncn1) [reply src/constants/UrlAndParameters.js:6]: re: 'would it be better to have an object with default keys and values that the UrlConstructor loops through?' Does sound a better idea

Comment (RichardAllum) [reply src/constants/UrlAndParameters.js:6]: I agree, but it doesn't have to be done in this ticket so I've accepted these changes

Comment (ncn1) [reply src/constants/UrlAndParameters.js:6]: However, I have attempted it, and it actually makes the tests especially much harder to understand - you are basically using virtually the same logic to construct the url you are expecting to get as is used in constructFunnelbackUrls.js itself, and it's not very readable.

Comment (ncn1) [reply src/constants/UrlAndParameters.js:6]: I don't want to rename smeta_contentType, as it reflects the actual parameter name, like all the others do. That name has been dictated by Funnelback.

**Pull request merged**